doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -145,13 +145,16 @@ schema :
 
 .. cmd:: Combined Scheme @ident from {+, ident}
 
-where each identᵢ after the ``from`` is a different inductive principle that must
-belong to the same package of mutual inductive principle definitions.
-This command generates the leftmost `ident` to be the conjunction of the
-principles: it is built from the common premises of the principles and
-concluded by the conjunction of their conclusions.
+  where each identᵢ after the ``from`` is a different inductive principle that must
```

Comment:
It would be best to reformulate this paragraph so that the description comes entirely after the command grammar.